core/src/test/kotlin/cz/ackee/localizer/plugin/core/util/FileUtils.kt

```diff
@@ -0,0 +1,12 @@
+package cz.ackee.localizer.plugin.core.util
+
+import java.io.File
+
+object FileUtils {
```

Comment:
hmm, i 'd probably remove the enclousing object and just define a public function. I think this is a relict from java world, kotlin code rarely uses this